src/main/java/com/zendesk/maxwell/producer/NatsProducer.java

```diff
@@ -0,0 +1,68 @@
+package com.zendesk.maxwell.producer;
+
+import com.zendesk.maxwell.MaxwellContext;
+import com.zendesk.maxwell.row.RowMap;
+import com.zendesk.maxwell.util.TopicInterpolator;
+import io.nats.client.Connection;
+import io.nats.client.Nats;
+import io.nats.client.Options;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+
+public class NatsProducer extends AbstractProducer {
+
+	private final Logger LOGGER = LoggerFactory.getLogger(NatsProducer.class);
+	private final Connection natsConnection;
+	private final String natsSubjectTemplate;
+
+	public NatsProducer(MaxwellContext context) {
+		super(context);
+		try {
+			List<String> urls = Arrays.asList(context.getConfig().natsUrl.split(","));
+
+			Options.Builder optionBuilder = new Options.Builder();
+
+			urls.forEach(optionBuilder::server);
+
+			Options option = optionBuilder.build();
+
+			this.natsConnection = Nats.connect(option);
+			this.natsSubjectTemplate = context.getConfig().natsSubject;
+
+		} catch (IOException | InterruptedException e) {
```

Comment:
can you limit the `try {} catch() ` block to only surround the parts of the code that might throw the exception?  I Imagine that only line 34 might throw exceptions?

Reply:
done in https://github.com/GlobalSport/maxwell/pull/4/commits/b1111d20a66d9bfad0e83caeb5cefc0c9866811f

Reply:
done in https://github.com/zendesk/maxwell/pull/1646/commits/d6e9b0077b5add7e9385ad05c2b097e63accc208